exercises/practice/rna-transcription/.approaches/introduction.md

```diff
@@ -0,0 +1,108 @@
+# Introduction
+
+There are at least three idomatic approaches to solving RNA Transcription in Go.
+You can use the `strings.Map()` function.
+Or you can use the `map` object to look up the letters.
+Or a `switch` statement can be used to translate the letters.
+
+## General guidance
+
+The key to solving RNA Transcription is to match the RNA value to the DNA value.
+
+## Approach: `strings.Map()` function
+
+```go
+// Package strand is a small library for returning the RNA complement of a DNA strand.
+package strand
+
+import "strings"
+
+// toRNA returns the RNA complement of the input DNA strand.
+func ToRNA(dna string) string {
+	return strings.Map(func(r rune) rune {
+		switch r {
+		case 'G':
+			return 'C'
+		case 'C':
+			return 'G'
+		case 'T':
+			return 'A'
+		case 'A':
+			return 'U'
+		}
+		return r
+	}, dna)
+}
+```
+
+For more information, check the [`strings.Map()` approach][approach-strings-map].
+
+## Approach: `map` object
+
+```go
+// Package strand is a small library for returning the RNA complement of a DNA strand.
+package strand
+
+var translate = map[rune]byte{
+	'G': 'C',
+	'C': 'G',
+	'T': 'A',
+	'A': 'U',
+}
+
+// ToRNA returns the RNA complement of the input DNA strand.
+func ToRNA(dna string) string {
+	rna := make([]byte, len(dna))
+	for i, c := range dna {
+		rna[i] = translate[c]
+	}
+	return string(rna)
+}
+```
+
+For more information, check the [`map` object approach][approach-map-object].
+
+## Approach: `switch` statement
+
+```go
+// Package strand is a small library for returning the RNA complement of a DNA strand.
+package strand
+
+func nucComp(nuc byte) byte {
+
```

Comment:
```suggestion
func nucComp(nuc byte) byte {
```

Reply:
Not sure what change is intended here.

Reply:
It was removing some whitespace that you already removed in a commit you added I think.